### DISCUSSION TITLE
totally disable cahing

### DISCUSSION THREAD

**Discussion by vehere-ccuMaksym** [shadowwalker/next-pwa#431] [Q&A]
How to totally disable all cachig?
If I enable "NetworkOnly" strategy for all routes browser still caching some of them and gives me some SW response...
And I tired to understand when and why it happens - it is almost random thing.

Tell me how to have PWA functions but with no cahing at all?

**Comment by Merott** [comment]
Have you found your answer yet? Looking for the same.

**Reply by vehere-ccuMaksym** [reply]
Unfortunately not :( Thinking on manual rewriting PWA and get rid of this extension.
